packages/feedback-components/src/feedback/text-visualizer.ts

```diff
@@ -40,16 +39,21 @@ function buildTags(
           active,
         });
 
-    tags.push({
-      color: tagStyle.backgroundColor,
+    const tag = {
+      color: tagStyle.color,
+      fontWeight: tagStyle.fontWeight,
```

Comment:
Try not to mess with font weights because it causes the text to reflow on each selection